src/mds/MDSMap.h

```diff
@@ -193,6 +193,7 @@ class MDSMap {
    */
 
   mds_rank_t max_mds; /* The maximum number of active MDSes. Also, the maximum rank. */
+  mds_rank_t standby_count_wanted;
```

Comment:
Currently I think that if the user decides he doesn't like the warning and wants to reset this to zero, we're going to go ahead and set it back to 1 in the cases where it's auto set during promote().  That's probably not desired behaviour -- what if instead we defaulted/initialized standby_count_wanted to -1, and then only auto-set it to 1 from -1.  If it was zero, we should assume that is a human-set value and not modify it for them.